### PR TITLE
feat(billing): add warning on payment method add page

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-4928
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/index.html [hunk 1]
```diff
@@ -1,6 +1,19 @@
 <div class="billing-payment-method-add">
     <div data-ovh-alert="billing_payment_method_add_alert"></div>
 
+    <oui-message class="mb-3" type="warning">
```
Comment (ganeshkumar1989) [inline packages/manager/apps/dedicated/client/app/billing/payment/method/add/index.html:4]: Should these messages come up only for US customers?

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -27,5 +27,7 @@
...
   "billing_payment_method_add_vantiv_recoverable_card_validation_number_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de validation de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_success": "Votre nouveau moyen de paiement a bien été ajouté.",
-  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}"
+  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}",
+  "billing_payment_method_add_alert_add_method": "Si vous rencontrez un problème pour ajouter un nouveau moyen de paiement, veuillez ajouter une exception à votre bloquer de publicités pour {{ hostname }}, ou veuillez vous connecter via le mode incognito de votre navigateur, puis réssayez. Nous n'afficherons jamais de publicités sur notre site web. Sachez que votre bloqueur de publicités peut interférer avec notre système de paiement.",
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json:31]:
```suggestion
  "billing_payment_method_add_alert_add_method": "Si vous rencontrez un problème pour ajouter un nouveau moyen de paiement, veuillez ajouter une exception à votre bloqueur de publicités pour {{ hostname }}, ou veuillez vous connecter via le mode incognito de votre navigateur, puis réessayez. Nous n'afficherons jamais de publicités sur notre site web. Sachez que votre bloqueur de publicités peut interférer avec notre système de paiement.",
```

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/index.html (hunk 1 above)
Comment (JeremyDec) [reply packages/manager/apps/dedicated/client/app/billing/payment/method/add/index.html:4]: I know the ticket is created based on US manager demand. But it concerns issue with ad blockers, which can be also applied to all regions.

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js [hunk 3]
```diff
@@ -28,6 +29,8 @@ export default class BillingPaymentMethodAddCtrl {
     this.loading = {
       redirecting: false,
     };
+
+    this.hostname = new URL(CORE_MANAGER_URLS.dedicated).hostname;
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js:33]: i would simply use window.location.hostname

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -27,5 +27,7 @@
...
   "billing_payment_method_add_vantiv_recoverable_card_validation_number_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de validation de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_success": "Votre nouveau moyen de paiement a bien été ajouté.",
-  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}"
+  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}",
+  "billing_payment_method_add_alert_add_method": "Si vous rencontrez un problème pour ajouter un nouveau moyen de paiement, veuillez ajouter une exception à votre bloqueur de publicités pour {{ hostname }}, ou veuillez vous connecter via le mode incognito de votre navigateur, puis réessayez. Nous n'afficherons jamais de publicités sur notre site web. Sachez que votre bloqueur de publicités peut interférer avec notre système de paiement.",
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json:31]: Not mandatory but i would sightly change the sentence to be more specific to the manager :
"nous n'afficherons jamais de publicités ~~sur notre site web~~ dans l'interface client" ?

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js (hunk 3 above)
Comment (JeremyDec) [reply packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js:33]: Hmm I did this, but for testing purpose, I add the constant to have the production correct hostname (and not staging link)

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json (hunk 4 above)
Comment (JeremyDec) [reply packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json:31]: Why not, but it is translated from given EN content, which has "our website" sentence

Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json:31]: @JeremyDec, maybe english content should be "our control panel"

File: packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js (hunk 3 above)
Comment (JeremyDec) [reply packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js]: Constant has been replaced by `window.location.hostname`

**Pull request merged**